manuals/1.0/en/reference.md

```
@@ -1,229 +1,194 @@
-    <doc format="markdown">Date of creation of the article expressed in ISO8601 format</doc>.
-</descriptor>
-The ``.
-The doc can be formatted (text|markdown|html|asciidoc) with format. If not specified, text is used.
-```
+## Description Formats
 
-### <a name="link">link</a>
+ALPS documents can be written in two formats:
 
-A link that links to a description of another resource.
+**XML Format**
```

Markdownlint

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)